java/server/src/org/openqa/selenium/grid/graphql/Session.java

```diff
@@ -46,7 +62,30 @@ public String getCapabilities() {
   }
 
   public String getStartTime() {
-    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm:ss");
-    return dtf.format(startTime);
+    return DATE_TIME_FORMATTER.format(startTime);
+  }
+
+  public URI getUri() {
+    return uri;
+  }
+
+  public String getNodeId() {
+    return nodeId;
+  }
+
+  public URI getNodeUri() {
+    return nodeUri;
+  }
+
+  public String getSessionDurationMillis() {
+    long duration = Duration.between(startTime, Instant.now()).toMillis();
+    return String.valueOf(duration);
   }
+
+  public org.openqa.selenium.grid.graphql.Slot getSlot() {
+    return new org.openqa.selenium.grid.graphql.Slot(slot.getId().getSlotId(),
```

Comment:
Nit: move `slot.getId().getSlotId()` to the next line, so the indentation is so gargantuan.